src/Host/Broker/Impl/Resources.resx

```diff
@@ -237,4 +237,7 @@
   <data name="Trace_UserProfileDirectory" xml:space="preserve">
     <value>User {0} profile directory: {1}</value>
   </data>
+  <data name="Exception_MaxAllowedUsers" xml:space="preserve">
+    <value>Broker has reached maximum allowed concurrent user limit.</value>
```

Comment:
I have made the change:
![image](https://cloud.githubusercontent.com/assets/3840081/19576583/585a8c2e-96c8-11e6-802c-1bd57d1dae54.png)